saleor/graphql/product/tests/test_product.py

```diff
@@ -2305,6 +2305,73 @@ def test_create_product(
     assert color_value_slug in values
 
 
+def test_create_search_product_by_description(
```

Comment:
Correct me if I am wrong. 
You are checking here if `create product mutation` creates data required to search right? Then you're checking if `search query` returns the expected data. 
Don't you think that it looks like two separate tests?
 - check if product created with the proper search data (like if the data exists in DB etc)
 - check if product search query returns expected results?

Reply:
I will create two seperate tests :)